docs/src/content/docs/de/reference/overrides.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: Komponenten-Ersetzung Referenz
-description: Ein Überblick über die Komponenten und Komponentenprops, die von Starlight Ersetzungen unterstützt werden.
+title: Ersetzung von Komponenten Referenz
```

Comment:
Related discussion for reference https://github.com/withastro/starlight/pull/2470/files/2b486d1b1fa4acf3f0c87da750f3ce219794f78e#diff-9185fd84aba7e9a14b394be96e2bd3562baac10c0285be708a3109c5cbe72802